scripts/get_counts.py

```diff
@@ -431,6 +437,25 @@ def ParseMemoryString(s):
     else:
         return (int(s), '')
 
+def DivideMemory(total, n):
+    (value, unit) = ParseMemoryString(total)
+    sub_memory = value / n
+    if sub_memory != float(value) / n:
+        if unit in ['K', '']:
```

Comment:
... also I think it would be better to have these statements 'elif', and then in the end have an 'else' that would throw an error, to more easily detect coding errors.